src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -994,11 +995,20 @@
                ExcludeEmptyDirectories="true"
                PackSymbolPackage="true"
                SymbolPackageOutputDirectory="$(SymbolPackageOutputPath)"
+               PackPrefixedSymbolPackage="$(PackPrefixedSymbolPackage)"
+               PrefixedSymbolPackageOutputDirectory="$(PrefixedSymbolPackageOutputPath)"
                AdditionalLibPackageExcludes="@(AdditionalLibPackageExcludes)"
                AdditionalSymbolPackageExcludes="@(AdditionalSymbolPackageExcludes)"
                Condition="'$(_SkipCreatePackage)' != 'true'"/>
-    <!-- Create a marker that records the path to the generated package -->
-    <WriteLinesToFile Lines="$(PackageOutputPath)$(Id).$(Version).nupkg;$(SymbolPackageOutputPath)$(Id).$(Version).symbols.nupkg"
+
+    <!-- Create markers that record the paths to the generated packages -->
+    <ItemGroup>
+      <OutputPackagePath Include="$(PackageOutputPath)$(Id).$(Version).nupkg" />
+      <OutputPackagePath Include="$(SymbolPackageOutputPath)$(Id).$(Version).symbols.nupkg" />
+      <OutputPackagePath Include="$(PrefixedSymbolPackageOutputPath)symbols.$(Id).$(Version).symbols.nupkg" />
```

Comment:
This last one should be conditional based on PackPrefixedSymbolPackage right? Perhaps even better, the NugetPack task could generate this item list for you rather than having the targets file do a parallel construction of the naming conventions. A typical place for bugs is when parallel computations of the same result get out of sync.